timesketch/frontend-ng/src/components/UploadForm.vue

```diff
@@ -133,11 +144,11 @@ limitations under the License.
             dialog = false
           "
         >
-          Close
+          Cancel
         </v-btn>
         <v-btn v-if="fileName" color="yellow darken-1" text @click="clearFormData()"> Clear Form </v-btn>
-
-        <v-btn color="green darken-1" text @click="submitForm()" :disabled="error.length > 0 || !fileName">
+        <b-field class="file is-primary" :class="{ 'has-name': true }"> </b-field>
```

Comment:
b-field, is this left over from the old UI?

Reply:
Yes, there will not be anymore there very soon with the chips I showed you